### PR TITLE
Keyboardissue

### PR DESCRIPTION
**Pull request by MariaRiosNavarro**
Fix Keyboard issues,
Now the sent messages and the pseudo imput are fixed in the 3 themes, so they can always be read. I fixed the "shift" button bug with a state, now it works.

Introduction to test: Go to settings in the NAV, choose the dark mode and go to the message or telefon app, choose the message icon and write and  enter

[Deployment](https://mobile-simulation-prop-7ni9yq29o-mariariosnavarro.vercel.app/)

### CODE REVIEW

File: components/MyKeyboard/index.js [hunk 1]
```diff
@@ -19,7 +22,24 @@ const StyledSendSMSContainer = styled.div`
...
+  }};
+  color: ${(props) => {
+    if (props.theme === "light") {
+      return "black";
+    } else if (props.theme === "dark") {
+      return "black";
+    } else if (props.theme === "custom") {
+      return "black";
+    }
+  }};
```
Comment (ghost) [inline components/MyKeyboard/index.js:42]: Here you forgot to adjust the actual assigned colors, I guess? Same for `background-color` above.

File: components/MyKeyboard/index.js [hunk 2]
```diff
@@ -43,12 +64,29 @@ const StyledSendedBackMessageContainer = styled.div`
...
+
+  color: ${(props) => {
+    if (props.theme === "light") {
+      return "black";
+    } else if (props.theme === "dark") {
+      return "black";
+    } else if (props.theme === "custom") {
+      return "black";
+    }
+  }};
```
Comment (ghost) [inline components/MyKeyboard/index.js:85]: Same here, every prop returns the same color

File: components/MyKeyboard/index.js [hunk 3]
```diff
@@ -60,12 +98,30 @@ const StyledSendedMessageContainer = styled.div`
...
+  color: ${(props) => {
+    if (props.theme === "light") {
+      return "black";
+    } else if (props.theme === "dark") {
+      return "black";
+    } else if (props.theme === "custom") {
+      return "black";
+    }
+  }};
```
Comment (ghost) [inline components/MyKeyboard/index.js:118]: same here

**Comment by ghost** [comment]
Hey Maria, I just realised that you are always returning the same color for every different prop.

**Comment by MariaRiosNavarro** [comment]
> Hey Maria, I just realised that you are always returning the same color for every different prop.

Exactly, I use useContext which makes everything change dynamically, in general, and I did not like the result, I want that in the case of the messages are always the same in the 3 theme. And there is no dynamic change, as it did not look good. If I only set the colour for example "background-color: white" the theme (light, dark and custom) will overwrite it in the Layout and useContext in _app. I don't want this to happen in these 3 cases, that's why I have to define them like this, so that the theme doesn't change them. It's not a bug

**Comment by ghost** [comment]
All good then! 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**